src/spec.ts

```diff
@@ -2,20 +2,21 @@ import { Keyring } from '@polkadot/api';
 import { cryptoWaitReady } from '@polkadot/util-crypto';
 const fs = require('fs');
 
-function nameCase(string) {
+function nameCase(string:string) {
 	return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
 // Get authority keys from within chainSpec data
-function getAuthorityKeys(chainSpec) {
-	if (chainSpec.genesis.runtime.runtime_genesis_config) {
-		return chainSpec.genesis.runtime.runtime_genesis_config.palletSession.keys;
-	}
+function getAuthorityKeys(chainSpec:ChainSpec) {
+	// if (chainSpec.genesis.runtime.frameSystem) {
+	// 	return chainSpec.genesis.runtime.frameSystem.palletSession.keys;
+	// }
+	// Why is this code trying to call palletSession on different fields?
```

Comment:
@shawntabrizi @pepyakin I had to comment the above part because the spec had `chainSpec.genesis.runtime.frameSystem` but palletSession was on the runtime level.
Could you give me a little bit more context on this code pls?

Reply:
oh I think it's backward compatibility. One of them is more recent, I don't know which though. It could be checked in a generated spec JSON file which created when launching polkadot-launch with a recent enough relay-chain node

Reply:
This code works with the binaries included in the zip file.
How do I generate more recent versions for relaychain in the same way you explained it for simple collator in https://github.com/paritytech/polkadot-launch/issues/52#issuecomment-772651055 ?

Reply:
pretty similar actually. It's

```
cargo build --release --features=real-overseer
```

Reply:
great thanks